packages/ComponentLibrary/src/components/Table/Sidebar/SidebarContent.tsx

```diff
@@ -21,6 +22,13 @@ export const SidebarContent: React.FC<SidebarContentProps> = ({
         </Typography>
       </Box>
     </Box>
+    <Box ml={2.5}>
```

Comment:
Is this a mock? Otherwise I would check the reusability of the component or pass the labels to constants.